Job launching after 0.76 seconds in submission.
Running python job.
Changed into dir /home/garth/FYP/product-taz/components/org.taz.core/src/main/python/autoencoder/experiments/gcsequence
Importing o.py
Running o.main()
I 2017-02-07 13:37:44 theanets.layers.base:371 layer Input "in": 1 inputs
I 2017-02-07 13:37:44 theanets.layers.base:207 layer LSTM "hid1": (in:out)1 -> 20, sigmoid, 1820 parameters
I 2017-02-07 13:37:44 theanets.layers.base:207 layer LSTM "hid2": (hid1:out)20 -> 20, sigmoid, 3340 parameters
I 2017-02-07 13:37:44 theanets.layers.base:207 layer Feedforward "out": (hid2:out)20 -> 1, linear, 21 parameters
I 2017-02-07 13:37:44 theanets.graph:94 network has 5181 total parameters
stateit 1
it 100000
I 2017-02-07 13:37:44 downhill.dataset:118 valid: 95 mini-batches from callable
I 2017-02-07 13:37:44 downhill.dataset:118 train: 283 mini-batches from callable
I 2017-02-07 13:37:44 theanets.graph:447 building computation graph
I 2017-02-07 13:37:44 theanets.losses:67 using loss: 1.0 * MeanSquaredError (output out:out)
I 2017-02-07 13:37:44 theanets.regularizers:711 regularizer: 0.812527924776 * GaussianNoise(('in:out',))
I 2017-02-07 13:37:54 downhill.base:389 -- patience = 5
I 2017-02-07 13:37:54 downhill.base:390 -- validate_every = 1
I 2017-02-07 13:37:54 downhill.base:391 -- max_updates = None
I 2017-02-07 13:37:54 downhill.base:392 -- min_improvement = 0.005
I 2017-02-07 13:37:54 downhill.base:393 -- max_gradient_norm = 0
I 2017-02-07 13:37:54 downhill.base:394 -- max_gradient_elem = 0
I 2017-02-07 13:37:54 downhill.base:395 -- learning_rate = 0.0001
I 2017-02-07 13:37:54 downhill.base:396 -- momentum = 0
I 2017-02-07 13:37:54 downhill.base:397 -- nesterov = True
I 2017-02-07 13:37:54 downhill.adaptive:220 -- rms_halflife = 14
I 2017-02-07 13:37:54 downhill.adaptive:221 -- rms_regularizer = 1e-08
I 2017-02-07 13:37:54 downhill.base:118 compiling evaluation function
I 2017-02-07 13:38:08 downhill.base:124 compiling RMSProp function
I 2017-02-07 13:57:29 downhill.base:232 validation 0 loss=3.442179 err=3.442179 *
I 2017-02-07 16:00:34 downhill.base:232 RMSProp 1 loss=1.646689 err=1.646689
I 2017-02-07 16:00:34 theanets.graph:626 /tmp/tmprPaF6M.tmp: saved model
I 2017-02-07 16:00:34 theanets.graph:643 /tmp/tmprPaF6M.tmp: loaded model
I 2017-02-07 16:17:39 downhill.base:232 validation 1 loss=1.144167 err=1.144167 *
I 2017-02-07 18:20:34 downhill.base:232 RMSProp 2 loss=0.946595 err=0.946595
I 2017-02-07 18:20:34 theanets.graph:626 /tmp/tmp2E0Bmk.tmp: saved model
I 2017-02-07 18:20:34 theanets.graph:643 /tmp/tmp2E0Bmk.tmp: loaded model
I 2017-02-07 18:37:54 downhill.base:232 validation 2 loss=0.534957 err=0.534957 *
I 2017-02-07 19:22:29 downhill.base:419 interrupted!
Traceback (most recent call last):
  File "/home/garth/FYP/Spearmint/spearmint/launcher.py", line 243, in launch
    result = python_launcher(job)
  File "/home/garth/FYP/Spearmint/spearmint/launcher.py", line 327, in python_launcher
    result = module.main(job['id'], params)
  File "../../omain.py", line 20, in main
    o=float(rnn.function(no_array,run_id=job_id))
  File "../../rnn.py", line 126, in function
    except StopIteration: pass
KeyboardInterrupt
Problem executing the function
Job failed in 20696.68 seconds.
(<type 'exceptions.KeyboardInterrupt'>, KeyboardInterrupt(), <traceback object at 0x7f6f24a7f1b8>)
